sycl/test/basic_tests/esimd/check_compile.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: %clangxx -fsycl -fsycl-explicit-simd -fsycl-device-only -fsyntax-only -Xclang -verify %s
```

Comment:
I think this test should be added to CLANG LIT not SYCL LIT.

Reply:
This test is temporary until we merge SIMD runtime library. I will remove it later, since it will be redundant.

Reply:
Please add TODO for that in comments to the test

Reply:
Done.